client/my-sites/plans-grid/components/actions.tsx

```diff
@@ -23,6 +24,8 @@ import { useCallback } from '@wordpress/element';
 import { localize, TranslateResult, useTranslate } from 'i18n-calypso';
 import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
 import { useManageTooltipToggle } from 'calypso/my-sites/plans-grid/hooks/use-manage-tooltip-toggle';
+import { useSelector } from 'calypso/state';
```

Comment:
Hello. We cannot use Calypso state/selectors anywhere inside `my-sites/plans-grid` folder. Everything in here is scheduled to migrate to an NPM package (under `/packags/plans-grid`) as early as next week perhaps. This is part of a general plans refactor we've been working on for several months now.

Happy to review any follow up changes, or feel free to reach out in Slack for additional details. It's weekend now, so this will have to wait until Monday.